src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1538,7 +1537,18 @@ namespace UnitarySystems {
         if (int(state.dataUnitarySystems->unitarySys.size()) == state.dataUnitarySystems->numUnitarySystems &&
             state.dataZoneEquip->ZoneEquipInputsFilled)
             setupAllOutputVars(state, state.dataUnitarySystems->numUnitarySystems);
-
+        for (std::size_t coilNUM = 1; coilNUM < state.dataUnitarySystems->unitarySys.size(); ++coilNUM) {
+            if (state.dataUnitarySystems->unitarySys[coilNUM].m_CoolingCoilType_Num == DataHVACGlobals::CoilDX_MultiSpeedCooling ||
+                state.dataUnitarySystems->unitarySys[coilNUM].m_HeatingCoilType_Num == DataHVACGlobals::CoilDX_MultiSpeedHeating) {
+                SetupEMSActuator(state,
+                                 "Coil Speed Control",
+                                 state.dataUnitarySystems->unitarySys[coilNUM].Name,
+                                 "Unitary System DX Coil Speed Level",
+                                 "[]",
+                                 state.dataUnitarySystems->unitarySys[coilNUM].m_EMSOverrideCoilSpeedNumOn,
+                                 state.dataUnitarySystems->unitarySys[coilNUM].m_EMSOverrideCoilSpeedNumValue);
+            }
+        }
```

Comment:
Well, 1) this belongs in setupAllOutputVars, not here. And 2), it should apply to all coil types that use `this->m_CoolingSpeedNum` so it just needs to be tested with the new coil model to make sure it works. MultiSpeed is different from VS so it likely won't work the first time. This conditional (if Multi*) does prevent that from causing problems with the release so that's good. A follow up issue could expand this to other coil types.